sycl/doc/extensions/PlatformContext/PlatformContext.adoc

```diff
@@ -0,0 +1,29 @@
+= SYCL(TM) Proposals: Platform Default Contexts
+James Brodman <james.brodman@intel.com>
+v0.1
+:source-highlighter: pygments
+:icons: font
+:y: icon:check[role="green"]
+:n: icon:times[role="red"]
+
+== Platform Default Contexts
+
+This extension adds the notion of a default SYCL context per SYCL platform. The default context for each platform contains all devices in the platform.
```

Comment:
It seems that the lifetime of the default context is not clear.
It can be important in situations, like: SYCL2020 says that in some cases unconsumed async exceptions should be passed to the `context` async handler during the context destruction.